v-next/hardhat/test/internal/builtin-plugins/network-manager/request-handlers/handlers/accounts/local-accounts.ts

```diff
@@ -451,6 +451,62 @@ describe("LocalAccountsHandler", () => {
   });
 
   describe("modifyRequest", () => {
+    describe("sending transactions to the null address", () => {
+      it("should succeed when the data field is not empty", async () => {
+        const tx = {
+          // null address: the "to" field is not required
```

Comment:
Given that this situation is kind of tricky, i think we should also add a test with `to: null`. That should behave exactly like this one (i.e. the zod validation probably needs to be updated).

Reply:
So for the `to` property the `null` value should be allowed and it should be handled as we do for `undefined`?

Reply:
Done [here](https://github.com/NomicFoundation/hardhat/pull/6017/commits/d08c6e08e410bcbec71e5c4cd7a3bce8869d3142)